R/dettl_config.R

```diff
@@ -46,7 +46,7 @@ dettl_locate_config <- function(path) {
 #'
 db_config_read_yaml <- function(filename, path) {
   info <- yaml_read(filename)
-  check_fields(info, filename, "source", "destination")
+  check_fields(info, filename, "source", c("destination", "vault_server"))
```

Comment:
It is probably worth validating the type of the `vault_server` field when present